src/loaders/lottie/tvgLottieModel.h

```diff
@@ -613,6 +613,28 @@ struct LottieComposition
 {
     ~LottieComposition();
 
+    LottieObject* findObject(const char* sid) {
```

Comment:
Suggestion: 
Essentially, ThorVG prefers simpler method names (if no any conflict) that consist of just a verb, like.
```
LottieObject* find(const char* sid);
```
As you can see, we can easily figure out the purpose of this method through method name and the return type value + the given parameter.